src/LessMsi.Gui/MainFormPresenter.cs

```diff
@@ -433,7 +434,7 @@ private void UpdateMSiTableGrid(Database msidb, string tableName)
 			if (msidb == null || string.IsNullOrEmpty(tableName))
 				return;
 
-			Status(string.Concat("Processing Table \'", tableName, "\'."));
+			Status(string.Concat($"{Strings.ProcessingTableStatus} \'", tableName, "\'."));

```

Comment:
minor, but ideally convert this to a `string.format` expression and put the placeholder into the string so that the translator can decide how to "concat" the text appropriately.